client_test.go

```diff
@@ -105,6 +110,41 @@ func (t *TestSuite) TestClient() {
 	t.NotEmpty(info.APIVersion)
 }
 
+// TestRoundTrip tests
+func (t *TestSuite) TestRoundTrip() {
+	client, err := NewClient(context.Background(), t.getRPCAddress(), t.token)
+	t.Require().NoError(err)
+	defer client.Close()
+
+	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+	defer cancel()
+
+	namespace, err := share.NewBlobNamespaceV0([]byte{1, 2, 3, 4, 5, 6, 7, 8})
+	t.Require().NoError(err)
+	t.Require().NotEmpty(namespace)
+
+	data := []byte("hello world")
+	blobBlob, err := blob.NewBlobV0(namespace, data)
+	t.Require().NoError(err)
+
+	// write blob to DA
+	txResponse, err := client.State.SubmitPayForBlob(ctx, math.NewInt(100000000), 200000000, []*blob.Blob{blobBlob})
```

Comment:
Why not `Blob.Submit`?

Reply:
Because SubmitPayForBlob is used by https://github.com/rollkit/rollkit/pull/986

Reply:
resolving this conversation to expedite. @Wondertan please create a follow-up, if you need any further change. thanks